liberty-managed/src/main/java/io/openliberty/arquillian/managed/WLPManagedContainer.java

```diff
@@ -202,32 +202,45 @@ public void start() throws LifecycleException
 
 			String javaVmArguments = containerConfiguration.getJavaVmArguments();
 
-            cmd.add(System.getProperty("java.home") + "/bin/java");
-            cmd.add("-Dcom.ibm.ws.logging.console.log.level=INFO");
-
-            // Only add JPMS options for Java 9+
-            if (!System.getProperty("java.specification.version").startsWith("1.")) {
-            		cmd.addAll(getJPMSOptions());
-            }
-            
-            if (!javaVmArguments.equals("")) {
-            	cmd.addAll(parseJvmArgs(javaVmArguments));
-         	}
-            cmd.add("-javaagent:lib/bootstrap-agent.jar");
-            cmd.add("-jar");
-            cmd.add("lib/ws-launch.jar");
-            cmd.add(containerConfiguration.getServerName());
-            
-            log.finer("Starting server with command: " + cmd.toString());
+//            cmd.add(System.getProperty("java.home") + "/bin/java");
+//            cmd.add("-Dcom.ibm.ws.logging.console.log.level=INFO");
+//
+//            // Only add JPMS options for Java 9+
+//            if (!System.getProperty("java.specification.version").startsWith("1.")) {
+//            		cmd.addAll(getJPMSOptions());
+//            }
+//
+//            if (!javaVmArguments.equals("")) {
+//            	cmd.addAll(parseJvmArgs(javaVmArguments));
+//         	}
+//            cmd.add("-javaagent:" + containerConfiguration.getWlpHome() + "bin/tools/ws-javaagent.jar");
+//            cmd.add("-jar");
+//            cmd.add(containerConfiguration.getWlpHome() + "bin/tools/ws-server.jar");
+//            cmd.add(containerConfiguration.getServerName());
+
+//            log.finer("Starting server with command: " + cmd.toString());
```

Comment:
lets keep this logging statement so we have trace of the actual command being passed into ProcessBuilder